src/reduceIndexed.js

```diff
@@ -0,0 +1,28 @@
+import { addIndex, reduce } from 'ramda';
+
+
+/**
+ * {@link http://ramdajs.com/docs/#reduce|R.reduce} function that more closely resembles Array.prototype.reduce.
+ * It takes two new parameters to its callback function: the current index, and the entire list.
+ *
+ * `reduceIndexed` implementation is simple : `
+ * const reduceIndexed = R.addIndex(R.reduce);
+ * `
+ * @func reduceIndexed
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.5.0|v2.5.0}
+ * @category List
+ * @typedef Idx = Number
+ * @sig ((a, b, Idx, [b]) => a) -> a -> [b] -> a
+ * @param {Function} fn The function to be called on every element of the input `list`
+ * @param {*} acc The accumulator value
+ * @param {Array} list The list iterate over
+ * @return {*} The final, accumulated value
+ * @see {@link http://ramdajs.com/docs/#addIndex|R.addIndex}, {@link http://ramdajs.com/docs/#reduce|R.reduce}
+ * @example const initialList = ['f', 'o', 'o', 'b', 'a', 'r'];
```

Comment:
Format this correctly as I've done here: https://github.com/char0n/ramda-adjunct/commit/318cf023d3aa2dd71cfd9bfc2ebc8eec48220058